### PR TITLE
Added bridge interface test

### PR DESCRIPTION
**Pull request by harish-24**
Test the ethernet bridge configuration in the linux kernel using brctl command

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

File: io/net/bridge.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+        cidr_cmd = "ip addr show %s | sed -nr 's/.*inet ([^ ]+)."\
+                   "*/\\1/p'" % self.host_interface
+        self.cidr = process.system_output(
+            '%s' % cidr_cmd, shell=True)
+        route_cmd = "route -n | grep %s | grep -w UG | awk "\
+                    "'{ print $2 }'" % self.host_interface
+        self.gateway = process.system_output(
+            '%s' % route_cmd, shell=True)
+        brd_cmd = "ip addr show %s | grep inet | grep brd | "\
```
Comment (narasimhan-v) [inline io/net/bridge.py]: `cidr_cmd`, `route_cmd`, etc variables can be `cmd`

File: io/net/bridge.py.data/bridge.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+host_interface: "eth0"
+peer_ip: "9.114.15.234"
```
Comment (narasimhan-v) [inline io/net/bridge.py.data/bridge.yaml]: Please leave these empty

**Comment by harish-24** [comment]
@narasimhan-v Pushed. Thanks

File: io/net/bridge.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            "awk '{ print $4 }'" % self.host_interface
+        self.broadcast = process.system_output(
+            '%s' % cmd, shell=True)
+
+    def test(self):
+        '''
+        Bridge Test
+        Set up the ethernet bridge configuration in the linux kernel
+        '''
+        process.system('brctl addbr br0', sudo=True)
```
Comment (PraveenPenguin) [inline io/net/bridge.py]: can we have a pram as `ignore_status=True` it help in case when command failed

Comment (PraveenPenguin) [inline io/net/bridge.py]: can we need check as bridge added or not

Comment (PraveenPenguin) [inline io/net/bridge.py]: this applicable all the places where `process.system` used

Comment (harish-24) [inline io/net/bridge.py]: @PraveenPenguin I think each step is as important as any other and if it fails, we should not be going to next stepand better to raise the `CmdError` exception.

Comment (harish-24) [inline io/net/bridge.py]: sure, will add it

Comment (PraveenPenguin) [inline io/net/bridge.py]: raising CmdError i think not good i think we have proper validation for every command if command output is failed (not matched with desired  output ) need to fail the test

**Comment by harish-24** [comment]
@PraveenPenguin Pushed with the changes. Thanks !

**Comment by PraveenPenguin** [comment]
@narasimhan-v thanks

**Pull request merged**